pkg/tidb/proxy.go

```diff
@@ -95,12 +98,32 @@ func (p *proxy) updateRemotes(remotes map[string]struct{}) {
 		}
 		return true
 	})
+	p.noAliveRemote.Store(false)
```

Comment:
Sorry for the slow response, why would this state change here, we don't know if any of remotes are alive yet.